clover-core/src/main/java/com/atlassian/clover/util/JavaEnvUtils.java

```diff
@@ -20,6 +27,7 @@ public class JavaEnvUtils {
             javaVersion = JAVA_8;
             Class.forName("java.lang.StackWalker");
             javaVersion = JAVA_9;
+            javaVersion = SourceVersion.latestSupported().toString().substring("RELEASE_".length());
```

Comment:
We should have auto-detection based on java runtime, i.e. find some class introduced in a specific Java release.

Reply:
I have changed the way to get the Java runtime version to use classes/methods introduced in each version.